File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/modal/camion.java
package progetto_terminal_marittimo.modal;

public class camion {
    private String targa;
    private String modello;

    public camion(String t, String m) {
        this.targa = t;
        this.modello = m;
    }

    public String getTarga() {
        return this.targa;
    }

    public String getModello() {
        return this.modello;
    }
}

File: pagine/js/camion.js
class Camion {
  constructor(targa, modello) {
    this.targa = targa;
    this.modello = modello;
  }

  getID() {
    return this.targa;
  }

  getModello() {
    return this.modello;
  }

  toString(){
    return "targa: "+this.targa+", modello: "+this.modello;
  }
}
